profiles/pci/handhelds/profiles.toml

```diff
@@ -97,10 +97,10 @@ vendor_ids = "1002"
 device_ids = "15bf"
 hwd_product_name_pattern = '(ROG Ally).*'
 priority = 6
-packages = 'inputplumber jupiter-hw-support steam cachyos-handheld mesa lib32-mesa vulkan-radeon lib32-vulkan-radeon opencl-rusticl-mesa lib32-opencl-rusticl-mesa rocm-opencl-runtime'
+packages = 'steam-powerbuttond-git inputplumber jupiter-hw-support steam cachyos-handheld mesa lib32-mesa vulkan-radeon lib32-vulkan-radeon opencl-rusticl-mesa lib32-opencl-rusticl-mesa rocm-opencl-runtime'
```

Comment:
BTW, do we really need to specify steam here? cachyos-handheld seems to be already depend on it.

Reply:
That is a different package.

Reply:
> That is a different package.

Can you explain in more detail?
```
[I] vasily@cachyos ~> expac -S '%D' -l "\n" cachyos-handheld | rg "^steam"
steam
steam-native-runtime
```